content/millennium/dstu2/devices/device.md

```diff
@@ -21,20 +42,47 @@ _Implementation Notes_
 
 * The [Device.status] modifier element is not supported and will not be returned.
 
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true)%>
+
 ### Parameters
 
  Name      | Required?         | Type          | Description
 -----------|-------------------|---------------|---------------------------------------------------------------------------
  `_id`     | This or `patient` | [`token`]     | The logical resource id associated with the Device. Example: `7891`
  `patient` | This or `_id`     | [`reference`] | The patient on whom the device is affixed. Example: `12345`
 
-### Response
+### Headers
 
-<%= headers status: 200, head: {Functionality: 'Search by patient'} %>
+<%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Device?patient=4478007
+
+#### Response
+
+<%= headers status: 200 %>
 <%= json(:dstu2_device_bundle) %>
-<%= headers status: 200, head: {Functionality: 'Search by id'} %>
+
+### Example Read by Ids
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Device?_id=15575768
+
+#### Response
+
+<%= headers status: 200%>
```

Comment:
Super nit: Some of the new templatings are missing a space before the trailing `%>`